.github/workflows/go.yml

```
@@ -13,25 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.x
-      uses: actions/setup-go@v2
-      with:
-        go-version: ^1.16
-
-    - name: Setup Node.js environment
-      uses: actions/setup-node@v2.1.2
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-    
-    - name: Setup Task
-      run: curl -sL https://taskfile.dev/install.sh | sh
-    
-    - name: Build using Taskfile.yml
-      run: ./bin/task build
-    
-    - name: Archive production artifacts
-      uses: actions/upload-artifact@v2
-      with:
-        name: compiled-application-amd64-ubuntu
-        path: out/*
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.16
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
```

You can use just `v2` to receive patches for this actions, at the moment the latest version is `2.1.5`:
```suggestion
        uses: actions/setup-node@v2
```